buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/IntellijPlugin.kt

```diff
@@ -27,6 +30,26 @@ fun Project.configureIntellijPlugin() {
     tasks.withType<BuildSearchableOptionsTask> { enabled = false }
   }
 
+  configure<ChangelogPluginExtension> {
+    version.set("v${project.version}")
+    path.set("${project.projectDir}/CHANGE-NOTES.md")
+    header.set(version)
+    headerParserRegex.set(Regex("v\\d+\\.\\d+\\.\\d+"))
+    unreleasedTerm.set("Unreleased")
+    groups.set(emptyList())
+  }
+
+  val changelog = extensions.getByType(ChangelogPluginExtension::class.java)
+
+  val changelogUnreleasedSection = changelog.getUnreleased().toText()
+    .replace("^- ".toRegex(), "<p>")
+    .replace("\n?$".toRegex(), "</p>")
+    .replace("\n- ", "</p>\n\n<p>")
+
+  tasks.register("getReleaseChangeLog") {
```

Comment:
For consistency with gradle-changelog-plugin, treat _changelog_ as a single word:
```suggestion
  tasks.register("getReleaseChangelog") {
```